bee-ternary/examples/simple.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+//
```

Comment:
```suggestion

```

Reply:
If i remove this line - there is no space between licence and running instructions

![Bildschirmfoto 2020-06-19 um 16 32 57](https://user-images.githubusercontent.com/10562055/85143820-8ceeb680-b24a-11ea-8040-ec95a8c46d78.png)

Reply:
There needs to be a blank line after the license

Reply:
Just remove the `//`, not the line :)